### DISCUSSION TITLE
Get owner and inventory name of enderchests :)

### DISCUSSION THREAD

**Discussion by Proemmis** [Zrips/CMI#9067] [Feature Requests]
Hello, it would be super great to add the possibility of knowing the owner and the inventory of the enderchests.

I would like to create a log system allowing you to know what the staff takes and deposits in the enderchest players, this can be useful in the event of an attempt to steal from the enderchests of banned players for example.

Thank you so much if this is added!

i posted an issue here : https://github.com/Zrips/CMI/issues/9024

**Comment by Proemmis** [comment]
bump this because i think that could be cool :)

**Comment by Proemmis** [comment]
bump this again ! i want it :)

**Comment by Proemmis** [comment]
bump

**Comment by Proemmis** [comment]
bump 2 month no answer

**Comment by Zrips** [comment]
It was generally already explained, CMI, when it comes to ender chest does extremely basic thing, entire code could be shrunk to single line as `player.openInventory(target.getEnderChest());` and due to how ender chest work inside minecraft itself this will not provide actual owner in its internal data, it's just basic limitation on how minecraft works, that isint convenient, but it is what it is.
